### ISSUE TITLE
Domain logic in worker

### ISSUE THREAD

**Issue by soumyaray** [Hyphasis/floofloo-api#43]
Seems like all the domain logic of your application is in a service object of a worker?  Should be in a domain object of its own.

https://github.com/Hyphasis/floofloo-api/blob/f004fc19bd316ee0dc03e850bba44be295e958f9/app/application/services/scheduled_worker.rb#L71

**Comment by tienyulin** [comment]
@soumyaray Yeah, the similarity part should be placed in domain, I didn't notice that. Thanks for your reminder.
